diff --git a/node_modules/vite-plugin-storybook-nextjs/dist/index.cjs b/node_modules/vite-plugin-storybook-nextjs/dist/index.cjs
index d9de231..d94ff15 100644
--- a/node_modules/vite-plugin-storybook-nextjs/dist/index.cjs
+++ b/node_modules/vite-plugin-storybook-nextjs/dist/index.cjs
@@ -2718,7 +2718,7 @@ var scopedResolve = (id) => {
   } catch (e) {
     scopedModulePath = require3.resolve(id);
   }
-  const idWithNativePathSep = id.replace(/\//g, posix.sep);
+  const idWithNativePathSep = id.replace(/\//g, path2.sep);
   if (scopedModulePath.endsWith(idWithNativePathSep)) {
     return scopedModulePath;
   }
